recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+############################################################################ ###
+# File rc.pvr
+# Copyright Copyright (C) 2013-2016 Renesas Electronics Corporation
+# License Strictly Confidential.
+#### ###########################################################################
+
+# Auto-generated for r8a7796_linux from rogueddk_1.7@4495854
```

Comment:
If it is autogenerated, why you are adding it?

Reply:
although it is autogenerated it is not part of the git repo we have and was originally distributed as part of the binary package

Reply:
Isn't it generated during DDK build?

Reply:
I was, wrong, it is generated - will do